superset-frontend/src/dashboard/components/SliceHeaderControls/index.tsx

```diff
@@ -573,15 +691,39 @@ const SliceHeaderControls = (props: SliceHeaderControlsPropsWithRouter) => {
         overlayStyle={dropdownOverlayStyle}
         trigger={['click']}
         placement="bottomRight"
+        visible={dropdownIsOpen}
       >
         <span
-          css={css`
+          css={() => css`
             display: flex;
             align-items: center;
           `}
           id={`slice_${slice.slice_id}-controls`}
           role="button"
           aria-label="More Options"
+          tabIndex={0}
+          onClick={() => {
+            toggleDropdown();
```

Comment:
I think `toggleDropdown` should go to `onVisibleChange` callback of `NoAnimationDropdown`

Reply:
nice, thanks for that tip! That worked great.